serve/mlc_serve/api/handler.py

```diff
@@ -49,14 +49,11 @@ def create_error_response(status_code: HTTPStatus, message: str) -> JSONResponse
 def _get_sampling_params(
     request: ChatCompletionRequest, model_artifact_config: ModelArtifactConfig
 ) -> SamplingParams:
-    sampling_params = SamplingParams(
-        # These params came from vllm
-        # TODO(amnalyshe): should they be put into mlc-llm batch serving ChatCompletionRequest?
-        # best_of=request.best_of,
-        # top_k=request.top_k,
-        # ignore_eos=request.ignore_eos,
-        # use_beam_search=request.use_beam_search,
-    )
+    sampling_params = SamplingParams()
```

Comment:
Can we simply do 
```
sampling_params = SamplingParams(
   presence_penalty=request.presence_penalty,
   ....
) 
```
Although we need to handle checks like `if request.presence_penalty is not None:`, I think this way we can make sure that these new values can go through the `__post_init__`. 
And I think `sampling_params.verify()` can also stay within `__post_init__` if this works.

Reply:
I tried that approach first, but I didn't want to repeat the default values specified in `sampling_params.py` here.